utils/hare-utils

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+CONSUL_ADDR=127.0.0.1
+CONSUL_PORT=8500
+
+wait4() {
```

Comment:
Quick grepping shows that this function is used in `hare-bootstrap` and `hare-node-join` scripts as well. Do you plan to refactor them in this PR?

Reply:
As per yesterday's discussion in DSU, there isn't an immediate need to refactor these scripts in this PR.

Reply:
Will you at least create Github issue to make sure that we don't forget to refactor that?

Reply:
Yes, created an issue https://github.com/Seagate/cortx-hare/issues/1519